pkg/generate/app/templatelookup.go

```diff
@@ -57,9 +61,13 @@ func (r TemplateSearcher) Search(terms ...string) (ComponentMatches, error) {
 						Score:       score,
 						Template:    template,
 					})
+					break
```

Comment:
If we had multiple terms we'd stop at the first level that had matches.
Not on the first match.

On Thu, Jan 21, 2016 at 11:58 AM, Ben Parees <notifications@github.com>
wrote:

> In pkg/generate/app/templatelookup.go
> <https://github.com/openshift/origin/pull/6670#discussion_r50428974>:
>
> > @@ -57,9 +61,13 @@ func (r TemplateSearcher) Search(terms ...string) (ComponentMatches, error) {
> >  						Score:       score,
> >  						Template:    template,
> >  					})
> > +					break
>
> i'm confused by this break. what if the term you "matched" an item to was
> not an exact match? don't you want to search on the other terms to see if
> there is a better match? (i'm not clear under what conditions we'd actually
> have multiple terms either, but...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6670/files#r50428974>.
>